aragog/parser.py

```diff
@@ -370,7 +370,7 @@ def get_heating(self, time: npt.NDArray | float) -> npt.NDArray | float:
             time: Time
 
         Returns:
-            Radiogenic heating as a float if time is a float, otherwise a numpy row array where
+            Radiogenic heating [W/kg] as a float if time is a float, otherwise a numpy row array where
```

Comment:
I rather think this function returns a non dimensional heating rate as it is used by the solver.

Reply:
I did some dimensional analysis on these quantities, and they return power per unit mass. This is eventually passed through to `dTdt()` in `solver.py` at which point it is converted to a heating rate (rate of temperature change over time).

They are non-dimensionalised though, so it is true that [W/kg] is a misnomer. I'll fix this.